fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -3019,6 +3024,14 @@ public boolean isEnableViewBasedMvRewrite() {
         return this.enableViewBasedMvRewrite;
     }
 
+    public void setEnableCBOViewBasedMvRewrite(boolean enableCBOViewBasedMvRewrite) {
+        this.enableCBOViewBasedMvRewrite = enableCBOViewBasedMvRewrite;
+    }
+
+    public boolean isEnableCBOViewBasedMvRewrite() {
```

Comment:
Why add this config?

Reply:
because maybe we need cbo view based mv rewrite sometime.
Add this session variable to enable it when necessary.